src/main/java/org/kainos/ea/validator/UserValidator.java

```diff
@@ -0,0 +1,14 @@
+package org.kainos.ea.validator;
+
+import org.kainos.ea.models.UserRequest;
+
+public class UserValidator {
```

Comment:
this needs unit tests

Reply:
added unit tests for valid/invalid email and password